api/src/shared/infrastructure/jobs/monitoring/MonitoredJobHandler.js

```diff
@@ -4,11 +4,11 @@ class MonitoredJobHandler {
     this.logger = logger;
   }
 
-  async handle(data, jobName) {
+  async handle({ data, jobName, jobId }) {
     let result;
     try {
       this.logJobStarting({ data, jobName });
```

Comment:
Suggestion: Faire aussi apparaître le jobId dans les log Starting et Failed pour faciliter le monitoring ? 

```suggestion
      this.logJobStarting({ data, jobId, jobName });
```

et dans logJobStarting (similaire pour lobJobFailed) : 
```
logJobStarting({ data, jobId, jobName }) {
    this.logger.info({
      data,
      jobId,
      handlerName: jobName,
      message: 'Job Started',
      type: 'JOB_LOG',
    });
  }
```